sdk/keyvault/azure-security-keyvault-administration/src/test/java/com/azure/security/keyvault/administration/KeyVaultCredentialPolicyTest.java

```diff
@@ -43,81 +43,72 @@ public class KeyVaultCredentialPolicyTest {
+
+        return callContextReference.get();
+    }
+
     @BeforeEach
     public void setup() {
         HttpRequest request = new HttpRequest(HttpMethod.GET, "https://kvtest.vault.azure.net");
         HttpRequest requestWithDifferentScope = new HttpRequest(HttpMethod.GET, "https://mytest.azurecr.io");
 
-        HttpPipelineCallContext plainContext = createMockContext(request, null);
+        Context bodyContextContext = new Context("KeyVaultCredentialPolicyStashedBody", BinaryData.fromString(BODY))
+            .addData("KeyVaultCredentialPolicyStashedContentLength", "21");
 
-        HttpPipelineCallContext differentScopeContext = createMockContext(requestWithDifferentScope, null);
+        Context bodyFluxContextContext = new Context("KeyVaultCredentialPolicyStashedBody", BODY_FLUX)
+            .addData("KeyVaultCredentialPolicyStashedContentLength", "21");
 
-        HttpPipelineCallContext testContext = createMockContext(request, null);
-
-        HttpPipelineCallContext bodyContext = createMockContext(request, callContext -> {
-            callContext.setData("KeyVaultCredentialPolicyStashedBody", BinaryData.fromString(BODY));
-            callContext.setData("KeyVaultCredentialPolicyStashedContentLength", "21");
-        });
-
-        HttpPipelineCallContext bodyFluxContext = createMockContext(request, callContext -> {
-            callContext.setData("KeyVaultCredentialPolicyStashedBody", BODY_FLUX);
-            callContext.setData("KeyVaultCredentialPolicyStashedContentLength", "21");
-        });
+        MockHttpResponse unauthorizedResponseWithoutHeader = new MockHttpResponse(
+            new HttpRequest(HttpMethod.GET, "https://azure.com"), 500);
 
         MockHttpResponse unauthorizedResponseWithHeader = new MockHttpResponse(
-            new HttpRequest(HttpMethod.GET, "https://kvtest.vault.azure.net"),
-            500,
-            new HttpHeaders().set(HttpHeaderName.WWW_AUTHENTICATE, AUTHENTICATE_HEADER)
-        );
+            new HttpRequest(HttpMethod.GET, "https://azure.com"), 500,
+            new HttpHeaders().set(HttpHeaderName.WWW_AUTHENTICATE, AUTHENTICATE_HEADER));
 
-        MockHttpResponse unauthorizedResponseWithoutHeader = new MockHttpResponse(
-            new HttpRequest(HttpMethod.GET, "https://kvtest.vault.azure.net"), 500);
+        MockHttpResponse unauthorizedResponseWithHeaderAndClaims = new MockHttpResponse(
+            new HttpRequest(HttpMethod.GET, "https://azure.com"), 500,
+            new HttpHeaders().set(HttpHeaderName.WWW_AUTHENTICATE, AUTHENTICATE_HEADER_WITH_CLAIMS));
 
         this.unauthorizedHttpResponseWithHeader = unauthorizedResponseWithHeader;
         this.unauthorizedHttpResponseWithoutHeader = unauthorizedResponseWithoutHeader;
-        this.callContext = plainContext;
-        this.differentScopeContext = differentScopeContext;
+        this.unauthorizedHttpResponseWithoutHeaderAndClaims = unauthorizedResponseWithHeaderAndClaims;
```

Comment:
Looks like this attribute has an authentication header:
```suggestion
        this.unauthorizedHttpResponseWithHeaderAndClaims = unauthorizedResponseWithHeaderAndClaims;
```